docs/topics/permissions.md

```diff
@@ -219,7 +219,7 @@ Roles represent a set of permissions attached to a group of users. Roles have na
 | tags?          | [role tags](/docs/topics/permissions#role-object-role-tags-structure) object  | the tags this role has                                                                                                          |
 | flags          | integer                                                                       | [role flags](/docs/topics/permissions#role-object-role-flags) combined as a [bitfield](https://en.wikipedia.org/wiki/Bit_field) |
 
-Roles without colors (`color == 0`) do not count towards the final computed color in the user list.
+Roles without colors (`colors.primary_color == 0`) do not count towards the final computed color in the user list.
```

Comment:
🤔 should it be both? `colour` isn't going away (even though it's deprecated), so shouldn't we document this for both values?

Reply:
The flip side of which is - if I'm a developer still using the deprecated API, I still want to know what's going on. I may not migrate to the new API for a while.

We might want two sentence though - it should still be clear that `color` is deprecated.

Maybe something like:

```
Roles without colors (`colors.primary_color == 0`) do not count towards the final computed color in the user list. If you are using the deprecated `color` property, it also acts in the same way.
```

Or something like that.